hw3/Task_1.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+//Task 1
+let x = 20;
+let y = 58;
+let z = 42;
+console.log(x + y + z);
+//Task 2
```

Comment:
Please delete all commented and redundant code in your tasks

Reply:
Done.